cg/meta/report/rnafusion.py

```diff
@@ -47,15 +48,16 @@ def get_sample_metadata(
             rin = self.lims_api.get_sample_rin(sample_id=sample.internal_id)
 
         return RnafusionSampleMetadataModel(
-            bias_5_3=sample_metrics.bias_5_3,
+            bias_5_3=sample_metrics.median_5prime_to_3prime_bias,
             duplicates=sample_metrics.pct_duplication,
             gc_content=sample_metrics.after_filtering_gc_content,
             input_amount=input_amount,
             insert_size=None,
             insert_size_peak=None,
-            mapped_reads=sample_metrics.reads_aligned
-            * 2
-            / sample_metrics.before_filtering_total_reads,
+            mapped_reads=get_mapped_reads_fraction(
+                mapped_reads=sample_metrics.read_pairs_examined * 2,
```

Comment:
maybe we can make `mapped_reads` a property on `sample_metrics` that does the caluclation and you can call `sample_metrics.mapped_reads` here instead. Would make the code more readable and understandable.